src/ir_Bosch.cpp

```diff
@@ -8,9 +8,11 @@
 //   Brand: Bosch,  Model: RG10A(G2S)BGEF remote
```

Comment:
Now that you've got a ir_Bosch.h file, and the "Supports" data is listed in that, you will need to remove the "Supports" comment section from the .cpp file. Otherwise it will fail one of our automated checks.
i.e. "Supports" info always goes into the .h file if it exists, if not, then into the .cpp file.

Reply:
Also, please add yourself to the Copyright section. You've added significant work too. ;-)